README.md

```diff
@@ -1,3 +1,165 @@
+
+## Configuration
+
+1. First, you need to install the app in your current Mattermost instance (refer to [Installation](#installation)), the ``/drive`` command should be available.
+2. Go to [Google Cloud Dashboard](https://console.cloud.google.com/home/dashboard) and create a new project (refer to [Create a Google Cloud Project](#create-a-google-cloud-project)).
+3. Return to Mattermost. 
+4. As a super admin user, run the ``/drive configure`` command.
+5. In the configuration modal, enter your Client ID, Client Secret and Service Account.
+    - Client ID and Client Secret: Used to generate the link to let the user connect with their Google Drive account.
+    - Service Account: Is the way the app will get the credentials to manage webhook notifications
+
+
+# Admin guide
+
+## Slash commands
+- ``/drive configure:``: This command will enable the connect and disconnect commands, that will allow the user to link with their Google Drive account. Client ID and Client Secret will be asked.
+
+
+# End user guide
+
+## Get started
+
+## Use ``/drive`` commands
+
+- ``/drive help``: This command will show all current commands available for this application.
+- ``/drive connect``: This command will create a new link, after clicked it will be redirected to select the Google account that will be used to execute all the available actions.
+- ``/drive create [docs | slide | sheets]``: This command will display a new modal where the data will be asked. 
+    - Title: Name of the file to be created 
+    - Message: Optional, only if a file is shared on the channel 
+    - File Access: Choose to share with members on the channel, anyone with the link or keep it private.
+- ``/drive notifications [start | stop]``: This command will start or stop the notifications. After a user has executed ``/drive connect`` command, notifications will start on the private channel with the user.
+
+
+## Channel header bindings
+
+- ``Save file on Drive:``: This option is available in any post, but validates that the selected post has any files on it. If the post
+has any files, a new modal will appear to let the user select the files that would like to be uploaded. After the action is done, a new post will appear as part of the thread to let you notified of the action.
+
+# Development
+
+## Manual installation
+
+*  Download the latest repository release.
+
+### Run the local development environment
+
+* You need to have installed at least node version 12 and maximum version 18. You can download the latest lts version of node for the required operating system here https://nodejs.org/es/download/
+
+*  Install libraries: ``cd`` to the project directory and execute ``npm install`` to download all dependency libraries.
+
+```
+$ npm install
+```
+
+*  Update the environment configuration file. The ``.env`` file must be modified or added to set the environment variables, it must be in the root of the repository.
```

Comment:
You should be able to provide env variables, and not require the `.env` file

Reply:
This can be achievable with the docker_compose.yml?